Sources/Commands/PackageTools/Format.swift

```diff
@@ -71,7 +69,7 @@ extension SwiftPackageTool {
             let args = [swiftFormat.pathString] + formatOptions + [packagePath.pathString] + paths
             print("Running:", args.map{ $0.spm_shellEscaped() }.joined(separator: " "))
 
-            let result = try TSCBasic.Process.popen(arguments: args)
+            let result = try await TSCBasic.Process.popen(arguments: args)
```

Comment:
I haven't looked, but I wonder if this is actually non-blocking

Reply:
Good catch, I'm cleaning that up on TSC side now.

Reply:
Wrapped that with a concurrent queue in https://github.com/apple/swift-tools-support-core/pull/450

Reply:
🤣 ask me if I have deadlocked the swift concurrency pool before...

Reply:
Do you want to use the safe_async wrapper until that lands?

Reply:
I've enabled auto-merge on that one, so I don't think there will be a significant amount of time when this PR is merged, but one in TSC isn't.